LeetCodeAssistant/LeetCodeProblemCache.swift

```diff
@@ -38,12 +38,14 @@ struct CodableLeetCodeProblem: LeetCodeProblem, Codable {
     var number: Int
     var difficulty: LeetCodeProblemDifficuly
     var title: String
+    var description: String
     
     init(from noncodable: LeetCodeProblem) {
         id = noncodable.id
         number = noncodable.number
         difficulty = noncodable.difficulty
         title = noncodable.title
+        description = ""
```

Comment:
Does this mean it's impossible to convert non-codable LeetCodeProblem instance into codable LeetCodeProblem instance with description?